exercises/largest-series-product/description.md

```diff
@@ -12,3 +12,5 @@ in the input; the digits need not be *numerically consecutive*.
 
 For the input `'73167176531330624919225119674426574742355349194934'`,
 the largest product for a series of 6 digits is 23520.
+
+For a series of zero digits, the largest product is 1 because 1 is the identity for multiplication.
```

Comment:
To discuss: Should a link be added? Candidates:

* https://en.wikipedia.org/wiki/Identity_element - note that this lists identities for operations other than multiplication as well.
* https://en.wikipedia.org/wiki/Empty_product - note that when reading the first paragraph you don't want to get mixed up between the additive identity 0 versus the multiplicative identity 1.

I'm not sure there should be a link here because I don't want to imply that reading the link is required for the student to solve this problem.

However, if that is a concern, we could avoid implying the link is required by instead saying "You don't have to know what a multiplicative identity is, but if you wanted to, you could read X"

Reply:
Wikipedia articles on maths topics are often rather overwhelming for people who aren't at least vaguely familiar with the topic. I think a link would help but perhaps there's a simpler site to link to.

Reply:
> You don't have to know what a multiplicative identity is, but if you wanted to, you could read X"

I like this suggestion because although technically correct, "multiplicative identity" is a fancy maths-y sounding term that may put people off, especially people that may be struggling with non-native English; and really instead of learning maths all the student needs to know is that for empty sets their implementation needs to return 1.

Reply:
Based on that concern (jargon may be offputting), I think it's good to say this anyway, even if we don't have a link to use. I'll try this:

"You don't need to know what a multiplicative identity is to solve this problem; it just means that multiplying a number by 1 gives you the same number."